src/Project/Project.router.ts

```diff
@@ -79,6 +79,15 @@ export class ProjectRouter extends Router {
       server.handleRequest(this.deleteProject)
     )
 
+    /**
+     * Update all projects with coords to null
+     */
+    this.router.delete(
+      '/projects/coords/:coords',
+      withAuthentication,
```

Comment:
aren't the following middlewares missing?

```
      withProjectExists,
      withProjectAuthorization
```

Reply:
As far as I know, those methods are used when you have the project ID to verify that the project exist. It also verify that you have the authorization to the project (by ID). This is done before doing the request, and since we don't have the project ID at that moment, we can't use these 

In this case, as we don't have the project ID, we update all the project associated with the eth_address that firm the request and removes the link to the coords. 

So it should be save because you won't be able to update others project unless you have their private key, and if you have it you could do all the calls anyways.

Reply:
I will defer to @nicosantangelo on this one since he has full context about that, from what I see in the code it may be a security issue but I'm lacking detailed understanding of this code to not seeing it as a risk.

To be more specific, I don't see _which_ project are you updating when you execute:
```ts
    await Project.update(
      { updated_at: new Date(), creation_coords: undefined },
      { creation_coords, eth_address }
    )
```

Reply:
Sure! From my understanding, I am updating the projects that have a link with the coords that you send to the API and are created from the same eth_address that are requesting the call